.github/workflows/npmpublish.yml

```diff
@@ -1,38 +1,39 @@
-name: "NPM Publish and GitHub tag"
+name: 'NPM Publish and GitHub tag'
 on:
   push:
     branches:
       - main
 
 jobs:
-  build:
+  publish:
```

Comment:
Split into `publish`, which runs `pnpm publish` and build, which creates GitHub tags